src/message.rs

```diff
@@ -155,12 +155,33 @@ impl Message {
     pub fn jetstream_message_info(
         &self,
     ) -> Option<crate::jetstream::JetStreamMessageInfo<'_>> {
-        let reply = self.reply.as_ref()?;
-        let mut split = reply.split('.');
-        if split.next()? != "$JS" || split.next()? != "ACK" {
+        const PREFIX: &'static str = "$JS.ACK.";
```

Comment:
Looking at https://github.com/nats-io/nats-architecture-and-design/blob/main/adr/ADR-15.md#js-ack and the go client https://github.com/nats-io/nats.go/blob/2ea8d393bbd4eb781462695533a0f1bf321ca7e5/js.go#L2416-L2451, it seems like it will always be at least this long. Are there other situations where it will just be `$JS.ACK` but without any tokens that follow?

Reply:
It will be at least that long but you can't depend on the '.' at the end, there may be additional information so just check for **$JS.ACK** as the prefix.